### PR TITLE
Make warnings in 5.9.0 release notes less scary; clarify it's a testing release

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

File: docs/appendices/release-notes/5.9.0.rst [hunk 1]
```diff
@@ -12,42 +12,22 @@ Released on 2024-10-09.
...
-    Please do a rolling upgrade from 5.8.x to 5.9.2 or higher instead.
-
-.. WARNING::
-
-    Tables that were created before CrateDB 4.x will not function with 5.x
-    and must be recreated before moving to 5.x.x.
+    5.9.0 was a testing release not promoted to stable due to `Known
+    issues`_.
+    The first stable version for the 5.9 line is :ref:`version_5.9.2`. Please
+    upgrade to 5.9.2 directly.
```
Comment (henrikingo) [inline docs/appendices/release-notes/5.9.0.rst:28]: What if this was immediately below the title? Below "Version 5.9.0". Replacing line 7. (Splitting hairs... This was never released.)

**Review by henrikingo — COMMENTED** [review]
Would also invite @geragray to review since you raised this.

**Review by seut — APPROVED** [review]

File: docs/appendices/release-notes/5.9.0.rst (hunk 1 above)
Comment (mfussenegger) [reply docs/appendices/release-notes/5.9.0.rst]: Moved it above the note

**Review by matriv — APPROVED** [review]
thx!

**Pull request merged**